cmd/vmonboarding.go

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package cmd
+
+import (
+	"errors"
+	"net"
+
+	"github.com/kubearmor/kubearmor-client/vm"
+	"github.com/spf13/cobra"
+)
+
+// vmPolicyAddCmd represents the command for vm onboarding
+var vmOnboardAddCmd = &cobra.Command{
+	Use:   "add",
+	Short: "onboard new vm onto nonk8s control plane",
+	Long:  `onboard new vm onto nonk8s control plane`,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return errors.New("requires a path to valid vm YAML as argument")
+		}
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		httpAddress := "http://" + net.JoinHostPort(HttpIP, HttpPort)
+		if err := vm.Onboarding("ADDED", args[0], httpAddress); err != nil {
+			return err
+		}
+		return nil
+	},
+}
+
+// vmOnboardDeleteCmd represents the command for vm offboarding
+var vmOnboardDeleteCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "offboard existing vm from nonk8s control plane",
+	Long:  `offboard existing vm from nonk8s control plane`,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return errors.New("requires a path to valid vm YAML as argument")
+		}
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		httpAddress := "http://" + net.JoinHostPort(HttpIP, HttpPort)
+		if err := vm.Onboarding("DELETED", args[0], httpAddress); err != nil {
+			return err
+		}
+		return nil
+	},
+}
+
+// vmListCmd represents the command for vm offboarding
+var vmListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "list configured vms from nonk8s control plane",
+	Long:  `list configured vms from nonk8s control plane`,
```

Comment:
```suggestion
	Short: "list configured VMs",
	Long:  `list configured VMs`,
```
VM list command will remain same regardless of k8s or non-k8s control plane.